docs/authentication/enterprise-connections/overview.mdx

```diff
@@ -41,7 +41,7 @@ The following IdPs are supported: Google Workspace and Microsoft Entra ID. For _
 
 Clerk prevents users linked to deprovisioned accounts in the OpenID provider from accessing the app.
 
-Within 10 minutes of a user being removed from the OpenID provider (e.g. [suspendeded](https://support.google.com/a/answer/33312?hl=en) or [deleted](https://support.google.com/a/answer/33314?hl=en) via Google Workspace, or [deleted](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-create-delete-users#delete-a-user) via Microsoft Entra), Clerk will recognize that the user has been deprovisioned and will revoke that user's existing sessions.
+Before creating a new session token for an EASIE user, Clerk verifies that the user has not been removed from the OpenID provider (e.g. [suspendeded](https://support.google.com/a/answer/33312?hl=en) or [deleted](https://support.google.com/a/answer/33314?hl=en) via Google Workspace, or [deleted](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-create-delete-users#delete-a-user) via Microsoft Entra), with a delay of up to 10 minutes. When deprovisioning is detected, Clerk revokes that user's existing sessions, and responds with 401 Unauthorized to any requests to retrieve a new session token.
```

Comment:
```suggestion
Before creating a new session token for an EASIE user, Clerk verifies that the user has not been removed from the OpenID provider (e.g. [suspended](https://support.google.com/a/answer/33312?hl=en) or [deleted](https://support.google.com/a/answer/33314?hl=en) via Google Workspace, or [deleted](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-create-delete-users#delete-a-user) via Microsoft Entra), with a delay of up to 10 minutes. When deprovisioning is detected, Clerk revokes that user's existing sessions and responds with 401 Unauthorized to any requests to retrieve a new session token.
```

Reply:
Aren't we able to detect disabled user accounts for Microsoft and deprovision? Did something change here?